standalone/inc/pup/PUPTrigger.h

```diff
@@ -19,17 +19,39 @@ typedef enum
    PUP_TRIGGER_PLAY_ACTION_CUSTOM_FUNC     // Call a custom function
 } PUP_TRIGGER_PLAY_ACTION;
 
+struct StateTrigger{
```

Comment:
we tried to stick with prefixing structs with PUP for example, also space after Trigger {